packages/module/src/AssistantMessageEntry/AssistantMessageEntry.tsx

```diff
@@ -0,0 +1,60 @@
+
+import React from 'react';
+import { Icon, Label, Split, SplitItem, TextContent } from '@patternfly/react-core';
+import { createUseStyles } from 'react-jss';
+import classnames from "clsx";
+
+import RobotIcon from '@patternfly/react-icons/dist/js/icons/robot-icon';
+
+const useStyles = createUseStyles({
+  chatbot: {
+    marginRight: "40px",
+  },
+  bubble: {
+    border: "1px solid var(--pf-v5-global--BackgroundColor--dark-400)",
+    borderRadius: "14px",
+    padding: "var(--pf-v5-global--spacer--sm) var(--pf-v5-global--spacer--md) var(--pf-v5-global--spacer--sm) var(--pf-v5-global--spacer--md)",
+    maxWidth: "100%",
+    wordWrap: "break-word",
+  },
+  label: {
+    // not working
+    "BorderColor": "var(--pf-v5-c-label--m-red__icon--Color)",
+  }
+})
+
+export const AssistantMessageEntry = ({ title, options }: { title: string; options?: string[] }) => {
+  const classes = useStyles();
+  return (
+    <div className="pf-v5-u-mb-md">
+      <Split className={classes.chatbot}>
+        <SplitItem>
+          <Icon size="lg" className="pf-v5-u-mr-sm pf-v5-u-pt-md">
+            <RobotIcon />
+          </Icon>
+        </SplitItem>
+        <SplitItem className={classnames(classes.bubble," pf-u-background-color-200")}>
```

Comment:
```suggestion
        <SplitItem className={classnames(classes.bubble," pf-v5-u-background-color-200")}>
```
without the v5, the className does not work

Reply:
Fixed